### PR TITLE
docs(Example): Update Stack to Volume Viewport With Annotations

### PR DESCRIPTION
**Pull request by jlopes90**
### Testing

Example Stack to Volume Viewport With Annotations
https://deploy-preview-1207--cornerstone-3d-docs.netlify.app/live-examples/stacktovolumewithannotations.html

#### PR

- [] My Pull Request title is descriptive, accurate and follows the
  semantic-release format and guidelines.

#### Code

- [] My code has been well-documented (function documentation, inline comments,
  etc.)

- [] I have run the `yarn build:update-api` to update the API documentation, and have
  committed the changes to this PR. (Read more here https://www.cornerstonejs.org/docs/contribute/update-api)

#### Public Documentation Updates

- [] The documentation page has been updated as necessary for any public API
  additions or removals.

#### Tested Environment

- [] "OS: 
- [] "Node version: 
- [] "Browser:
  

[blog]: https://circleci.com/blog/triggering-trusted-ci-jobs-on-untrusted-forks/
[script]: https://github.com/jklukas/git-push-fork-to-upstream-branch

### CODE REVIEW

**Comment by jlopes90** [comment]
https://github.com/cornerstonejs/cornerstone3D/issues/1045

I already tested it and still the same problem.

File: packages/core/src/utilities/convertStackToVolumeViewport.ts [hunk 1]
```diff
@@ -43,7 +46,11 @@ async function convertStackToVolumeViewport({
   const { id, element } = viewport;
   const viewportId = options.viewportId || id;
 
-  const imageIds = viewport.getImageIds();
+  let imageIds = viewport.getImageIds();
+  imageIds = imageIds.map((imageId) => {
+    const imageURI = utilities.imageIdToURI(imageId);
+    return `${VOLUME_LOADER_SCHEME}:${imageURI}`;
+  });
```
Comment (jlopes90) [inline packages/core/src/utilities/convertStackToVolumeViewport.ts:53]: I found what's missing or the problem and tested it and it works fine. But what do you think?

**Comment by jlopes90** [comment]
I don't find the problem. Anyway, just an example improvement.

**Comment by jlopes90** [comment]
@sedghi ready

**Review by sedghi — APPROVED** [review]

**Pull request merged**